weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -831,16 +444,11 @@ const PanelPlotConfigInner: React.FC<PanelPlotProps> = props => {
     () => (
       <>
         <Button size="mini" onClick={resetConfig}>
-          {'Reset & Automate Plot'}
+          Reset & Automate Plot
         </Button>
         <Button size="mini" onClick={condense}>
-          {'Condense'}
+          Condense
         </Button>
-        {/* {weavePythonEcosystemEnabled && (
-          <Button size="tiny" onClick={exportAsCode}>
-            {'Export as Code'}
-          </Button>
-        )} */}
```

Comment:
We should keep this in a commented out form in case we want to re-enable it later on